apps/examples/wifi_manager_sample/rt_perf.c

```diff
@@ -126,15 +131,23 @@ void
 _print_stability(rt_stability *stab)
 {
 	rt_stability_stat *s = &stab->stat;
-	printf("             %-11d%-8d%-8d%-7d         :%-d\n", s->count, s->pass, s->fail, s->skip, s->result);
+	if (s->result == DA_TC_PASS) {
+		printf("             %-11d%-8d%-8d%-7d         :SUCCESS\n", s->count, s->pass, s->fail, s->skip);
+	} else {
+		printf("             %-11d%-8d%-8d%-7d         :FAILURE\n", s->count, s->pass, s->fail, s->skip);
+	}
 }
 
 
 void
 _print_performance(rt_performance *perf)
 {
 	rt_performance_stat *p = &perf->stat;
-	printf("             %-11d%-8d%-8d%-8d%-8d:%-d\n", p->count, p->max/1000, p->min/1000, p->sum/1000, p->fail, p->result);
+	if (p->result == DA_TC_PASS) {
```

Comment:
Do we need to define this as ```DA_XX``` in public?

Reply:
@sunghan-chang All DA_XX has been modified to WIFIMGR_STRESS.